free-2.md

```diff
@@ -5,40 +5,86 @@
 This section describes who the project would serve and why a language might be a
 good way to meet their needs.
 
+This language would be for people with menstrual cycles to track and generate 
+information about their periods. A language could be a useful way to address this, 
+because the way people manage and track their periods is very personal. By using
+a language, this might allow for more flexibility and personal customization.
+
 
 ### What's the need?
 _What need is met by your idea? Who are you helping? What is that person's
 experience like now? What would their experience be like if you could help 
 them?_
 
+Having information tracking a person's period and the related symptoms is
+important health information. This data is useful to have during doctor's
+visits and when planning everyday life. Theoretically, this could help
+anyone with a period (cis-women, trans-men, and others). Right now,
+there are a few apps on the market that help with tracking this sort
+of thing, but a lot of them are frustrating to use or extremely feminized.
+If I could provide this tool, it might be more comfortable to use in
+more gender neutral settings and gives more options for people with
+periods in general.
+
 
 ### Why a language?
 _Why is a DSL appropriate for your user(s)? How does it address the need?_
 
+Periods are very personal. Using a DSL would allow for a lot more flexibility
+in tracking in than sticking to the format and layout of an app. This also
+could help with generate records which is useful when interacting with doctors.
 
 ### Why you?
 _What excites you about this idea? How did you come up with it?_
 
+I think this idea is exciting because having good care and knowledge of 
+menstrual cycles is very important to personal health and productivity.
+For cis-women in developing countries, this tends to be a factor that 
+holds them back (such as in going to school or dealing with the stigma).
+While I can't create a tool that address all of those problems, I like
+the idea of coming up with a tool that can help with some of the issues.
+I came up with this by googling some of the biggest problems currently 
+facing cis-women.
 
 ### Domain
 _Describe the project's domain in five words._
 
+Tracking menstrual cycles and symptoms.
+
 
 ### Interface (syntax)
 _How might the user interact with the language? What does programming look 
 like? Why is this the right way to interact with the problem domain?_ 
 
+The user could interact with the language in ways like tracking data in 
+an excel sheet or writing out simple commands. Programming looks like
+entering current menstrual state, date, time, and other relevant data about
+mood and health. I don't know if this is the right way to interact with 
+the problem domain to be honest. If I were to choose this project, I would
+start by interviewing people and seeing how they would prefer to interact
+with the language.
 
```

Comment:
I feel like you put this very well. I was trying to think what you meant by "writing out simple commands." It seems obvious that you would use the language to store data about how you are feeling and what is happening in your body, but that you would also need simple commands to allow you to retrieve data and perhaps create graphical representations of it so that you can see long term patterns and notice if anything unusual is happening. These sorts of graphs could be very useful to show doctors if you need to go in because something unusual is occurring.